src/main/java/com/rcv/ResultsWriter.java

```diff
@@ -428,32 +436,46 @@ private void generateSummaryJson(String outputPath,
 
     // root outputJson dict will have two entries:
     // results - vote totals, transfers, and candidates elected / eliminated
-    // TODO: add needed config info
     // config - global config into
     HashMap<String, Object> outputJson = new HashMap<>();
+    // config will contain contest configuration info
+    HashMap<String, Object> configData = new HashMap<>();
+    // add config header info
+    configData.put("Contest", config.getContestName());
+    configData.put("Jurisdiction", config.getContestJurisdiction());
+    configData.put("Office", config.getContestOffice());
+    configData.put("Date", config.getContestDate());
+    configData.put("Threshold", winningThreshold.toString());
```

Comment:
nit: `toString` not needed here?

Reply:
surprisingly, this works

Reply:
Counter-nit: `winningThreshold.toString()` is less confusing to me as a reader because it explicitly converts the `BigDecimal` to a `String`.

Reply:
On second thought, doesn't this need to be a String to correctly preserve BigDecimal representation?  Think of this as serializing an object, not just for display purposes.

Reply:
I assumed all of these were just getting converted to strings when it outputs them anyway, so it makes no difference. Anyway, whatever works.